__tests__/app.test.js

```diff
@@ -17,6 +18,29 @@ afterAll(() => {
   return db.end();
 }); //After each test end connection to db.
 
+describe("/api",() => {
+  // Happy Path
+  it("200: Should return an object describing all of the available endpoints.", () => {
+    return request(app) // send request to app.
+      .get("/api") //GET req to endpoint.
+      .expect(200)
+      .then(({ body }) => {
+        const { allValidEndpoints } = body // Deconstruct allValidEndpoints 
+        expect(allValidEndpoints).toEqual(endPoints) // check recieved to match endpoints.json contents.
+      })
+  })
+});
+
+   // Sad Path Test
+   it("404: Should return 'Path Not Found' error message if incorrect endpoint is requested (eg. typo)", () => {
```

Comment:
You already have a bad path/nonexistent endpoint test added previously so this isn't necessary, as any bad path should have the same behaviour 🔍